tests/pipeline/test_pipeline_trace.py

```diff
@@ -368,7 +368,8 @@ def test_trace_telemetry() -> None:
                 # dummy has empty fingerprint
                 assert event["properties"]["destination_fingerprint"] == ""
         # we have two failed files (state and data) that should be logged by sentry
-        assert len(SENTRY_SENT_ITEMS) == 2
+        # TODO: make this work
```

Comment:
I am not sure why this is not working anymore, can you point me to the place in the code where these sentry events should be raised?

Reply:
what this test was doing: failing two jobs. those generated two log messages on error level. those messages were sent to sentry.

you changed the code so we do not log anything when messages are failed or dummy was changed in a way to not fail the packages when `FAIL_PROB` is set.

this test should work again

Reply:
Alright, now I get it. I fixed the dummy config and I was logging 2 exceptions per failed job, which I have changed to only log the one from the loader, now it is good again.